### PR TITLE
BaSP-MR-161: setup of wdio/allure & set test-ids

### PR DESCRIPTION
**Pull request by JulianRiedinger7**
- Initial Automation setup using WebDriverIO with Jasmine as framework. (No Babel)
- Reporter configuration using Allure reports.
- Test-ids added in React components to facilitate the selection of elements with WebDriverIO.
- eslint configuration modified to accept WebDriverIO variables as globals.

### CODE REVIEW

File: src/Components/Admins/Form/index.jsx [hunk 1]
```diff
@@ -160,10 +160,10 @@ function AdminsForm() {
             />
           </div>
           <div className={styles.container_button}>
-            <Button action={reset} text="Reset" classNameButton="deleteButton" />
```
Comment (jramire5) [inline src/Components/Admins/Form/index.jsx:163]: Why did you delete this style?

**Review by gabiholl — APPROVED** [review]

File: src/Components/Members/Form/index.jsx [hunk 2]
```diff
@@ -199,10 +199,10 @@ const MemberForm = ({ match }) => {
             />
           </div>
           <div className={styles.reset_button}>
-            <Button action={reset} text="Reset" classNameButton="deleteButton" />
```
Comment (jramire5) [inline src/Components/Members/Form/index.jsx:202]: here as well

File: src/Components/Shared/Button/button.module.css [hunk 3]
```diff
@@ -22,7 +22,7 @@
 }
 .cancelButton {
   color: #fdfdfd;
-  background-color: #403F3F;
```
Comment (jramire5) [inline src/Components/Shared/Button/button.module.css:25]: here

File: src/Components/SuperAdmins/Form/index.jsx [hunk 4]
```diff
@@ -132,10 +132,10 @@ const SuperAdminsForm = () => {
                 register={register}
               />
               <div className={styles.reset_button}>
-                <Button action={reset} text="Reset" classNameButton="deleteButton" />
```
Comment (jramire5) [inline src/Components/SuperAdmins/Form/index.jsx:135]: add the className here again as well

File: src/Components/Admins/Form/form.module.css [hunk 5]
```diff
@@ -91,3 +91,7 @@
 .container_button {
   text-align: center;
 }
```
Comment (WalterArielPavicich) [inline src/Components/Admins/Form/form.module.css]: this class should be deleted from here
```suggestion

```

File: src/Components/Admins/Form/form.module.css [hunk 6]
```diff
@@ -91,3 +91,7 @@
...
+
+.container_button button {
+  background-color: grey;
+}
```
Comment (WalterArielPavicich) [inline src/Components/Admins/Form/form.module.css]: same here

File: src/Components/Admins/Profile/index.jsx [hunk 7]
```diff
@@ -20,7 +20,7 @@ function AdminProfile() {
   const [showConfirmModal, setShowConfirmModal] = useState(false);
   const { show, message, state } = useSelector((state) => state.toast);
   const loading = useSelector((state) => state.admins.isPending);
-  const defaultAdmin = useSelector((state) => state.admins.defaultAdmin || '');
+  const defaultAdmin = useSelector((state) => state.admins.defaultAdmin);
```
Comment (WalterArielPavicich) [inline src/Components/Admins/Profile/index.jsx]: this should not be modified
```suggestion
  const defaultAdmin = useSelector((state) => state.admins.defaultAdmin || '');
```

File: src/Components/Classes/ClassForm/index.jsx [hunk 8]
```diff
@@ -245,7 +245,7 @@ function ClassForm() {
             />
           </div>
         </form>
-        <Button classNameButton="deleteButton" text="Reset" action={reset} />
+        <Button classNameButton="deleteButton" text="RESET" action={reset} />
```
Comment (WalterArielPavicich) [inline src/Components/Classes/ClassForm/index.jsx]: this should not be modified
```suggestion
        <Button classNameButton="deleteButton" text="Reset" action={reset} />
```

File: src/Components/Members/Form/form.module.css [hunk 9]
```diff
@@ -60,6 +60,10 @@
   text-align: center;
 }
 
+.reset_button button {
+  background-color: grey;
+}
+
```
Comment (WalterArielPavicich) [inline src/Components/Members/Form/form.module.css]: this class should be deleted from here
```suggestion

```

File: src/Components/SuperAdmins/Form/form.module.css [hunk 10]
```diff
@@ -36,3 +36,7 @@
 .container_button, .reset_button {
   text-align: center;
 }
+
+.reset_button button {
+  background-color: grey;
+}
```
Comment (WalterArielPavicich) [inline src/Components/SuperAdmins/Form/form.module.css]: this class should be deleted from here
```suggestion

```

**Review by santimenendez98 — APPROVED** [review]

**Review by WalterArielPavicich — APPROVED** [review]

**Review by jramire5 — APPROVED** [review]
Good Job!

**Review by WalterArielPavicich — APPROVED** [review]
